src/main/docker/gitlab/gitlab.yml

```diff
@@ -0,0 +1,36 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# GitLab base service
+# ----------------------------------------------------------------------------------------------------------------------
+
+services:
+    gitlab:
+        container_name: artemis-gitlab
+        extra_hosts:
+            - "host.docker.internal:host-gateway"
+        build: .
+        platform: amd64
```

Comment:
```suggestion
        platform: linux/amd64
```

Reply:
Ah I was super confused why this `linux/amd64` is necessary at all. Just retested building this completely without the platform tag on a `aarch64` system and had issues building Angular (host got stuck). Then just with the `linux/amd64` I got build errors which look like I'm using the wrong arch. Then I installed QEMU and tada it works. Probably MacOS also uses some kind of emulation.